pkg/router/router.go

```diff
@@ -218,6 +218,8 @@ func registerEndpoints(r endpoints.Resource, container *restful.Container) {
 	wsv1.Route(wsv1.POST("/{namespace}/rerun").To(r.RerunPipelineRun))
 	wsv1.Route(wsv1.GET("/{namespace}/ingress").To(r.GetIngress))
 	wsv1.Route(wsv1.GET("/{namespace}/endpoints").To(r.GetEndpoints))
+	wsv1.Route(wsv1.GET("/{namespace}/dashboardversion").To(r.GetDashboardVersion))
```

Comment:
as mentioned on the [previous PR](https://github.com/tektoncd/dashboard/pull/935#discussion_r369502053) would it make sense to include this in the properties endpoint so we can get them all together instead of requiring separate calls for each value?

Reply:
pasting slack message here for reference:

if you add the versions into the existing `/v1/properties` endpoint you won’t need to register new URLs or worry about namespace
see:
- https://github.com/tektoncd/dashboard/blob/a0866ee7b14bbc81bd97a5ccea0bce4c8bd6bcd9/pkg/router/router.go#L268
- https://github.com/tektoncd/dashboard/blob/a0866ee7b14bbc81bd97a5ccea0bce4c8bd6bcd9/pkg/endpoints/cluster.go#L161

we could have a response something like:

```
{
  installNamespace: 'tekton-pipelines',
  versions: {
    dashboard: '0.5.0',
    pipeline: '0.10.1',
    triggers: '0.2.1'
  }
}
```

or have each version at the same level as the InstallNamespace

Having them all in one API simplifies things from the front end, as we only need to make 1 request on page load to get the info, and it’s unlikely to change.

Reply:
Have added these to the properties endpoint